### PR TITLE
feat(services): use ws endpoints to pull service metrics update table info

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-539

### CODE REVIEW

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery  preview 937d87a0-2391-4f54-8dbb-e263f2b05c0e | [test-old-status](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/937d87a0-2391-4f54-8dbb-e263f2b05c0e/services/general) |
 /qovery  preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery  preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> There are two hard problems in computer science: naming things, cache invalidation and off-by-1 errors

File: libs/shared/ui/src/lib/components/inputs/input-text-small/input-text-small.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ export function InputTextSmall(props: InputTextSmallProps) {
     <div data-testid="input-small-wrapper" className={`${className} ${classNameError}`}>
       {(error || warning) && errorMessagePosition === 'left' && (
         <Tooltip content={error || warning || ''} align="center" side="top">
-          <div data-testid="warning-icon-left" className="flex item-center">
+          <div data-testid="warning-icon-left" className="flex items-center">
```
Comment (ctjhoa) [inline libs/shared/ui/src/lib/components/inputs/input-text-small/input-text-small.tsx:56]: This is not related to the PR, just a type that I've spotted randomly

File: libs/domains/services/feature/src/lib/pods-metrics/pods-metrics.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+export function PodsMetrics({ environmentId, serviceId }: PodsMetricsProps) {
+  const { data = [], isLoading } = useMetrics({ environmentId, serviceId })
+
+  if (isLoading) {
+    return (
+      <div className="w-full flex items-center justify-center">
+        <LoaderSpinner className="w-6" />
+      </div>
```
Comment (RemiBonnet) [inline libs/domains/services/feature/src/lib/pods-metrics/pods-metrics.tsx:18]: Instead of use this one, I think we can use `<Skeleton />` for each cell, should be more elegant, wdyt ?

**Review by RemiBonnet — CHANGES_REQUESTED** [review]
Great, thanks @ctjhoa master of WS 🥇 

- You have a shrink with the right column, maybe it need a fixed width
Capture d’écran 2023-08-31 à 11 05 31

- 
Capture d’écran 2023-08-31 à 11 05 31

File: libs/domains/services/feature/src/lib/pods-metrics/pods-metrics.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    )
+  }
+
+  if (data === null) {
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/domains/services/feature/src/lib/pods-metrics/pods-metrics.tsx:22]: I can't do that because I want to distinguish `null` from `undefined`

File: libs/domains/services/feature/src/lib/pods-metrics/pods-metrics.tsx (hunk 2 above)
Comment (ctjhoa) [reply libs/domains/services/feature/src/lib/pods-metrics/pods-metrics.tsx]: https://github.com/Qovery/console/pull/796/commits/3c8bfd06afee79fd63c60123fb498b75f6bba737

File: libs/domains/services/feature/src/lib/pods-metrics/pods-metrics.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        <Table.Row>
+          <Table.ColumnHeaderCell className="font-medium w-1/2">Instance name</Table.ColumnHeaderCell>
+          <Table.ColumnHeaderCell className="font-medium">RAM usage</Table.ColumnHeaderCell>
+          <Table.ColumnHeaderCell className="font-medium">vCPU</Table.ColumnHeaderCell>
+          <Table.ColumnHeaderCell className="font-medium">Storage</Table.ColumnHeaderCell>
+        </Table.Row>
+      </Table.Header>
+      <Table.Body>
+        {data?.length ? (
+          data?.map((podMetrics) => (
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/domains/services/feature/src/lib/pods-metrics/pods-metrics.tsx]: Yep this one could be removed

File: libs/domains/services/feature/src/lib/pods-metrics/pods-metrics.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+      <Table.Header>
+        <Table.Row>
+          <Table.ColumnHeaderCell className="font-medium w-1/2">Instance name</Table.ColumnHeaderCell>
+          <Table.ColumnHeaderCell className="font-medium">RAM usage</Table.ColumnHeaderCell>
+          <Table.ColumnHeaderCell className="font-medium">vCPU</Table.ColumnHeaderCell>
+          <Table.ColumnHeaderCell className="font-medium">Storage</Table.ColumnHeaderCell>
+        </Table.Row>
+      </Table.Header>
+      <Table.Body>
+        {data?.length ? (
```
[the remark this answers is not in the document]
Comment (ctjhoa) [reply libs/domains/services/feature/src/lib/pods-metrics/pods-metrics.tsx]: Nope this one must handle `undefined` case

**Review by RemiBonnet — APPROVED** [review]
LGTM 👍 I just add a `shrink-0` for the about section https://github.com/Qovery/console/pull/796/commits/2581a3d504c3159c8bca972f913a230f9ae8c371
